src/libs/util/tests/tarixarchive_test.cpp

```
@@ -0,0 +1,267 @@
+
+    QFile m_file;
+    z_stream m_zs = {};
+    qint64 m_rawPos = 0;
+};
+} // namespace
+
+class TarixArchiveTest : public QObject
+{
+    Q_OBJECT
+
+private slots:
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)